packages/p2p/src/components/my-ads/my-ads-row-renderer.jsx

```diff
@@ -56,9 +60,20 @@ const MyAdsRowRenderer = observer(({ row: advert, setAdvert }) => {
                     is_left_swipe
                     right_hidden_component={
                         <React.Fragment>
-                            <div className='p2p-my-ads__table-popovers__edit' onClick={onClickEdit}>
-                                <Icon custom_color='var(--general-main-1)' icon='IcEdit' size={16} />
-                            </div>
+                            {is_advert_active ? (
+                                <div className='p2p-my-ads__table-popovers__edit' onClick={onClickEdit}>
+                                    <Icon custom_color='var(--general-main-1)' icon='IcEdit' size={16} />
+                                </div>
+                            ) : (
+                                <></>
+                            )}
+                            {!is_advert_active ? (
+                                <div className='p2p-my-ads__table-popovers__edit' onClick={onClickEdit}>
+                                    <Icon custom_color='var(--general-main-1)' icon='IcEdit' size={16} />
+                                </div>
+                            ) : (
+                                <></>
+                            )}
```

Comment:
```suggestion
                            {is_advert_active ? (
                                <div className='p2p-my-ads__table-popovers__edit' onClick={onClickEdit}>
                                    <Icon custom_color='var(--general-main-1)' icon='IcEdit' size={16} />
                                </div>
                            ) : (
                                <></>
                            )}
                            {!is_advert_active ? (
                                <div className='p2p-my-ads__table-popovers__edit' onClick={onClickEdit}>
                                    <Icon custom_color='var(--general-main-1)' icon='IcEdit' size={16} />
                                </div>
                            ) : (
                                <></>
                            )}
```

HMMMM these two are the same components. If `is_advert_active` is true, it will show the first one and not the second. If `is_advert_active` is false, it will not show the first but show the second. But both of these are the same components so we do not need the condition in the first place? 😆

Reply:
Oops.. Will refactor the code